server/controllers/recommendation/ppcs/checkBookingDetailsController.test.ts

```diff
@@ -22,6 +26,8 @@ const PRISON_OFFENDER_TEMPLATE = {
   lastName: prisonOffenderLastName,
   facialImageId: 1234,
   dateOfBirth: '1970-03-15',
+  agencyId: 'KLN',
+  agencyDescription: 'The Kyln',
```

Comment:
This is another field I've seen in a few places. Should we have this as codified test data somewhere?

Reply:
No, these come from a system external to our team - there is an endpoint to get all the values we are interested in, but the API for the endpoint doesn't pre-define what the full set of possible values is.